src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -1668,391 +1668,930 @@ static void vehicle_sprite_2_4(
     }
 }
 
-// 6D476C
-static void VehiclePitchUp25(
+static void vehicle_sprite_2_5(
```

Comment:
are you able to provide better naming than 2_5?

Reply:
I named it like that to be consistent with the way the other vehicle paint functions were named but I can change it if you want.

Reply:
@spacek531 started renaming some of this to be easier to understand I would like to continue that.

Reply:
OK I've renamed the vehicle paint functions